project/src/main/world/overworld-exit.gd

```diff
@@ -26,8 +26,10 @@ export (String) var instructor_spawn_id: String
 
 # 'true' if the player is currently overlapping the exit
 var _player_overlapping := false
+var _player_exiting := false
```

Comment:
Document this new field.

Reply:
Added comment, and expanded `_player_overlapping` comment to make the distinction more clear